ocs_ci/deployment/aws.py

```diff
@@ -258,12 +259,16 @@ def deploy(self, log_cli_level='DEBUG'):
             with open("./upi_on_aws-install.sh", "w") as fd:
                 fd.write(data)
 
-            sys.path.append(self.upi_script_path)
-            full_path = os.getcwd()
+            # ensure oc is on PATH for UPI install script
+            test_env = os.environ.copy()
+            path = test_env.get('PATH')
+            oc_path = os.path.join(constants.TOP_DIR, 'bin')
```

Comment:
@petr-balogh if we do that we will have to move where we determine the `oc_path` to before we start changing the `cwd` to satisfy the install shell script. We can pull it from config and use the absolute path to cover the jenkins case, that's not a problem. We were just seeing issues with that previously because it is a relative path and we are changing the `cwd` to allow the shell script to work properly.

Reply:
@petr-balogh you're actually right, after adding the `bin_dir`'s absolute path to the PATH, I no longer needed to add a custom env to subprocess. This will be in the next update.

Reply:
Done